verification/cpp/suite/test_unionant.cpp

```diff
@@ -116,13 +116,21 @@ TEST(UnionantTests, union_value_copy_ctor)
 TEST(UnionantTests, union_value_move_ctor)
 {
     using ValueType = uavcan::_register::Value_1_0;
-    const std::string hello_world{"Hello World"};
+    const char* hello_world{"Hello World"};
     ValueType         a{};
     // verify that empty is the default such that our emplace of string (next line) is actually changing the
     // variant's value type.
     ASSERT_NE(nullptr, ValueType::VariantType::get_if<ValueType::VariantType::IndexOf::empty>(&a.union_value));
     uavcan::primitive::String_1_0& a_result = a.union_value.emplace<ValueType::VariantType::IndexOf::string>(
-        uavcan::primitive::String_1_0{std::vector<std::uint8_t>(hello_world.begin(), hello_world.end())});
+        uavcan::primitive::String_1_0
+        {
+            {
+                reinterpret_cast<const unsigned char*>(hello_world),
+                reinterpret_cast<const unsigned char*>(&hello_world[11]),
+                11
+            }
```

Comment:
Speaking of strings, perhaps we should prioritize this: https://github.com/OpenCyphal/specification/issues/51 ? The confusion between arrays of numbers, strings, and arrays of bytes is getting in the way occasionally.